### PR TITLE
Feat/top epoch

### PR DESCRIPTION
**Pull request by faheelsattar**
adds support to query all data for top 5 epochs along with which slots are shielded

### CODE REVIEW

File: backend/internal/data/sql/queries/shutter_explorer.sql [hunk 1]
```diff
@@ -119,3 +119,37 @@ FROM (
 ) AS latest_events
 WHERE 
     is_registeration = TRUE;
+
+-- name: QuerySlotAndValidatorDataByEpoch :many
+SELECT 
+    pd.public_key, 
+    pd.validator_index, 
+    pd.slot,
+	pd.epoch,
```
Comment (fredo) [inline backend/internal/data/sql/queries/shutter_explorer.sql:128]: indentation typo?

File: backend/internal/data/sql/queries/shutter_explorer.sql [hunk 2]
```diff
@@ -119,3 +119,37 @@ FROM (
...
+        vrm.validity = 'valid'
+        AND vs.status = 'active_ongoing'
+    ORDER BY 
+        vrm.validator_index, 
+        vrm.created_at DESC
+) AS le
+ON 
+    pd.validator_index = le.validator_index
+WHERE 
+    pd.epoch IN (SELECT UNNEST($1::BIGINT[]))
```
Comment (fredo) [inline backend/internal/data/sql/queries/shutter_explorer.sql:153]: regarding my comment in the other PR: 

could you do something like `WHERE pd.slot BETWEEN X AND Y`?

File: backend/internal/data/sql/queries/shutter_explorer.sql (hunk 1 above)
Comment (faheelsattar) [reply backend/internal/data/sql/queries/shutter_explorer.sql]: i think it was github code viewer issue, but we have gotten rid of epoch anyways now

File: backend/internal/data/sql/queries/shutter_explorer.sql (hunk 2 above)
Comment (faheelsattar) [reply backend/internal/data/sql/queries/shutter_explorer.sql]: yep, updated the query

**Review by fredo — APPROVED** [review]
looks good to me!

**Pull request merged**